### PR TITLE
Adding other Roundabout-Exits

### PR DESCRIPTION
**Pull request by Niklas04**
Adding function to show Arrow-Value (for debugging)
Adding LeaveRoundaboutSharpLeft
Adding LeaveRoundaboutEasyLeft
Adding LeaveRoundaboutSharpRight
Adding LeaveRoundaboutEasyRight
Adding LeaveRoundaboutAsDirection

### CODE REVIEW

File: GoogleMaps_HUD/gmaps_hud/src/main/java/sky4s/garminhud/app/NotificationMonitor.java [hunk 1]
```diff
@@ -599,6 +599,9 @@ private void processArrow(Arrow arrow) {
             case LeaveRoundaboutEasyRight:
                 garminHud.SetDirection(eOutAngle.EasyRight, eOutType.RightRoundabout, eOutAngle.EasyRight);
                 break;
+            case LeaveRoundaboutAsDirection:
+                garminHud.SetDirection(eOutAngle.Down, eOutType.RightRoundabout, eOutAngle.Down);
```
Comment (skyforcetw) [inline GoogleMaps_HUD/gmaps_hud/src/main/java/sky4s/garminhud/app/NotificationMonitor.java]: What is LeaveRoundaboutAsDirection arrow like?
Could you show some image?

Comment (skyforcetw) [reply GoogleMaps_HUD/gmaps_hud/src/main/java/sky4s/garminhud/app/NotificationMonitor.java]: I remove LeaveRoundaboutAsUp, replace with LeaveRoundaboutAsDirection, keep same rule as other arrow.

Comment (Niklas04) [reply GoogleMaps_HUD/gmaps_hud/src/main/java/sky4s/garminhud/app/NotificationMonitor.java]: LeaveRoundaboutAsDirection is this symbol:
image
(Means: Use Roundabout to make a turn)

**Comment by skyforcetw** [comment]
Got it, I miss understand it.

File: GoogleMaps_HUD/gmaps_hud/src/main/java/sky4s/garminhud/app/NotificationMonitor.java (hunk 1 above)
Comment (skyforcetw) [reply GoogleMaps_HUD/gmaps_hud/src/main/java/sky4s/garminhud/app/NotificationMonitor.java]: Got it, I miss understand it.

**Pull request merged**